packages/edit-site/src/components/layout/index.js

```diff
@@ -175,10 +175,14 @@ export default function Layout( { route } ) {
 					{ isMobileViewport && areas.mobile && (
 						<div className="edit-site-layout__mobile">
 							{ canvasMode !== 'edit' && (
-								<SiteHub
-									ref={ toggleRef }
-									isTransparent={ isResizableFrameOversized }
-								/>
+								<SidebarContent routeKey={ routeKey }>
```

Comment:
@jsnajdr if you have any thoughts here.

Reply:
Another side effect of wrapping in `SidebarContent` is that the site hub receives extra padding, so there's a jump horizontally when we dig into the lower levels of the hierarchy:

https://github.com/WordPress/gutenberg/assets/14988353/97c95039-8f78-4822-b28b-0581332699fa